src/util/ConfigManager/ConfigManager.h

```diff
@@ -46,6 +46,32 @@ concept Validator =
     std::regular_invocable<Func, const ParameterTypes&...> &&
     std::same_as<std::invoke_result_t<Func, const ParameterTypes&...>, bool>;
 
+// Simple struct, that holds an error message. For use as the return type of
+// invocable object, that fullfill `ExceptionValidator`.
+struct ErrorMessage {
+  const std::string message_;
+
+  // Constructor.
+  ErrorMessage(std::string_view message) : message_{message} {}
+
+  // Move constructor. A move assignment operator is not possible, because
+  // `message_` is const.
+  ErrorMessage(ErrorMessage&&) = default;
```

Comment:
See above, make the member mutable, then the implicit defaults are correct.

Reply:
In hindsight, having that operator didn't make much sense. Deleted it in [0d21125](https://github.com/ad-freiburg/qlever/pull/1086/commits/0d21125a376e2b720be595cea86d1f294f2f7ed4).